core/src/main/java/feast/core/service/JobCoordinatorService.java

```diff
@@ -81,6 +85,14 @@ public JobCoordinatorService(
     this.jobProperties = feastProperties.getJobs();
     this.specPublisher = specPublisher;
     this.groupingStrategy = groupingStrategy;
+    this.featureSetSubscriptions =
+        feastProperties.getJobs().getCoordinator().getFeatureSetSelector().stream()
+            .map(JobProperties.CoordinatorProperties.Selector::toSubscription)
+            .collect(Collectors.toList());
+    this.blacklistedStores = feastProperties.getJobs().getCoordinator().getBlacklistedStores();
+    this.currentVersion = feastProperties.getVersion();
+    this.jobLabels = new HashMap<>(feastProperties.getJobs().getCoordinator().getJobSelector());
+    this.jobLabels.put("version", this.currentVersion);
```

Comment:
should `this.currentVersion` be a getter?

Reply:
this is private final property, not sure how getter could be useful?